src/modelversionstatus.hpp

```diff
@@ -48,29 +48,15 @@ inline const std::string& ModelVersionStateToString(ModelVersionState state) {
 
 enum class ModelVersionStatusErrorCode : int {
     OK = 0,
-    // CANCELLED = 1,
+    ERROR = 1,
     UNKNOWN = 2,
-    // INVALID_ARGUMENT = 3,
-    // DEADLINE_EXCEEDED = 4,
-    // NOT_FOUND = 5,
-    // ALREADY_EXISTS = 6,
-    // PERMISSION_DENIED = 7,
-    // UNAUTHENTICATED = 16,
-    // RESOURCE_EXHAUSTED = 8,
-    FAILED_PRECONDITION = 9,
-    // ABORTED = 10,
-    // OUT_OF_RANGE = 11,
-    // UNIMPLEMENTED = 12,
-    // INTERNAL = 13,
-    // UNAVAILABLE = 14,
-    // DATA_LOSS = 15,
-    // DO_NOT_USE_RESERVED_FOR_FUTURE_EXPANSION_USE_DEFAULT_IN_SWITCH_INSTEAD_
-    //    = 20
+    FAILED_PRECONDITION = 3,
```

Comment:
This corresponds I think to:
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/protobuf/error_codes.proto
I am not sure if this is exposed in our API but we might be breaking compatibility (current or in future) with TFS.

Reply:
Then ErrorCode::Error just does not make any sense to me. Error code should be indicating what happened wrong if possible. If not we should have UNKNOWN?